src/common/configuration/ConfigService.js

```diff
@@ -76,7 +76,7 @@
         },
         sources: [
           {
-            'url': (location.host + '/geoserver/web/'),
+            'url': (location.host + '/geoserver/wms/'),
```

Comment:
Saw this PR after I submitted my PR which addresses this issue also. A couple of notes:

- Looks like there is a declaration of `var serverLocation` a few lines above which also includes the current protocol.
- The function declares $location no I'm not sure if location.host will reference the same arg. Maybe instead of using `location.host` here, replace it for `serverLocation`.